### ISSUE TITLE
Fails to parse non-specific node tags

### ISSUE THREAD

**Issue by svend** [go-yaml/yaml#75]
``` go
package main

import (
    "fmt"
    "log"

    "gopkg.in/yaml.v2"
)

var data = `
- "12"
- 12
- ! 12
`

func main() {
    m := make(map[interface{}]interface{})

    err := yaml.Unmarshal([]byte(data), &m)
    if err != nil {
        log.Fatalf("error: %v", err)
    }
    fmt.Printf("--- m:\n%v\n\n", m)
}
```

Results in `2015/02/10 12:47:50 error: yaml: line 3: did not find URI escaped octet`.

According to the [YAML 1.2 non-specific tag documentation](http://yaml.org/spec/1.2/spec.html#id2784064), this YAML should parse to:

``` yaml
%YAML 1.2

---
!!seq [
  !<tag:yaml.org,2002:str> "12",
  !<tag:yaml.org,2002:int> "12",
  !<tag:yaml.org,2002:str> "12",
]
```

**Comment by a-chernykh** [comment]
This bugs me too :+1:

**Comment by hashmap** [comment]
Pull request has been submitted https://github.com/go-yaml/yaml/pull/133

**Comment by emosbaugh** [comment]
+1

**Comment by geofffranks** [comment]
It's been a year, and this hasn't been merged, or really discussed. Is there a timeline for getting this fixed?

**Comment by nmiyake** [comment]
+1 -- ran into this issue as well and confirmed that the PR that is currently open resolves this issue. This seems strictly better/more correct from a correctness standpoint. Is there any reason that this issue/PR isn't being advanced?

**Comment by niemeyer** [comment]
Yeah, sorry about that. As noticed elsewhere, this last year of mine was specially busy with work elsewhere. But we've just made the critical release a couple of weeks ago and I'm slowly picking up on the work I left behind. go-yaml is next in line.

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
Just ran into this too, any updates?

**Comment by rogpeppe** [comment]
Fixed by #103
